tsconfig.json

```diff
@@ -3,11 +3,11 @@
     "target": "es2019",
     "module": "commonjs",
     "moduleResolution": "node",
-    "lib": ["es2017", "dom"],
-    "types": ["node", "@wdio/globals/types"],
+    "lib": ["esnext", "dom"],
+    "types": ["node"],
     "declaration": true,
     "strict": true,
-    "skipLibCheck": true,
+    "skipLibCheck": false,
```

Comment:
Minor, not blocking: I think this setting be removed it because `false` is the default value.